src/main/java/models/Team.java

```diff
@@ -0,0 +1,45 @@
+package models;
+
+import java.util.Arrays;
+
+public class Team {
+
+    private Player[] players;
+    private int score;
+    private int wins;
+    private String teamName;
+
+    public Team(Player[] players, String teamName){
+        this.players = players;
+        this.score = calculateScore();
+        this.wins = 0;
+        this.teamName = teamName;
+    }
+
+    private int calculateScore(){
+        return Arrays.stream(players).mapToInt(player -> player.getType().getScore()).sum();
+    }
+
+    public int getTeamScore(){
+        return score;
+    }
+
+    public int getTotalWins(){
+        return wins;
+    }
+
+    public String getTeamName(){
+        return teamName;
+    }
+
+    public void addNewWin(){
+        wins++;
+    }
+
+    public void setPlayers(Player[] players) {
```

Comment:
I feel like a team's players should be known in advance, so they should not be able to be edited, but I'm not super sure on this one